apps/rush/src/RushCommandSelector.ts

```diff
@@ -7,6 +7,11 @@ import * as rushLib from '@microsoft/rush-lib';
 
 type CommandName = 'rush' | 'rushx' | undefined;
 
+export interface IExceuteOptions {
```

Comment:
Spelling.

Also, can't this be imported from the (non-version-selected) **rush-lib** dependency?